californium-core/src/main/java/org/eclipse/californium/core/CoapServer.java

```diff
@@ -147,7 +147,7 @@ public CoapServer(final NetworkConfig config, final int... ports) {
 		}
 		
 		// resources
-		this.root = createRoot();
+		this.root = new RootResource();
```

Comment:
Why should we limit  custom behavior to  `GET /` request only ?

E.g. For  `Bootstrap delete` in LWM2M, you need a custom behavior on `DETELE /`  :)